app/src/main/java/com/sdpteam/connectout/event/nearbyEvents/filter/CalendarEventFilter.java

```diff
@@ -0,0 +1,17 @@
+package com.sdpteam.connectout.event.nearbyEvents.filter;
+
+import com.sdpteam.connectout.event.Event;
+import com.sdpteam.connectout.profile.Profile;
+
+public class CalendarEventFilter implements EventFilter{
+    String profileId;
+
+    public CalendarEventFilter(String profileId){
+        this.profileId = profileId;
+    }
+
+    @Override
+    public boolean test(Event event) {
+        return event.getParticipants().contains(profileId);
```

Comment:
Maybe I'm wrong but here should not we as well check if the profileId is or the organiser?

Reply:
By default, organizers are part of an event so this should be good